src/app/ReviewService.Abstractions/ReviewSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ReviewService.Abstractions;
+
+/// <summary>
+/// The review prompt settings used for prompt conditions.
+/// </summary>
+public record ReviewSettings
+{
+	public int PrimaryActionCompletedCount { get; set; }
+
+	public int SecondaryActionCompletedCount { get; set; }
+
+	public int ApplicationLaunchedCount { get; set; }
```

Comment:
I think you want to use **Launch** instead of **Launched** because Launch is also a **noun**, whereas Launched is a verb.